### PR TITLE
Chore: fix commands

### PR DESCRIPTION
**Pull request by albertandking**
1. fix schedule.yml

### CODE REVIEW

File: .github/workflows/schedule.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ jobs:
         run: python -m pip install tox
 
       - name: Run ruff
-        run: tox -e format
+        run: tox -e format -- --fix
```
Comment (edtechre) [inline .github/workflows/schedule.yml:22]: Won't this suppress the format error, since this fix will never be applied to the source code?

Comment (albertandking) [reply .github/workflows/schedule.yml:22]: Yes，i will never be apply to tests and just set the same format like main.yml

Comment (edtechre) [reply .github/workflows/schedule.yml:22]: Please see https://github.com/edtechre/pybroker/commit/a62e437432fdf9f8e5e7d7ae0478c14d01a0e765

- Adds "check" arg in to Tox commands, and removes these from the .yml files
- Uses --diff along with --fix for format

Comment (albertandking) [reply .github/workflows/schedule.yml:22]: Ok, it work!

**Pull request closed without merging**